generate.sh

```diff
@@ -52,47 +55,63 @@ do
   completion+="\n  $fullCommand)"
   completion+="\n    _command_args=("
 
-  delimitedFlags=$(jq -r '. | select((.command == "'$command'") and (.topic == "'$topic'")) | .flags | .[] | .name + "\t" + .description + "\t" + .type + "\t" + .char' commands-display.json)
-  
+  delimitedFlags=$(jq -r '. | select((.command == "'$command'") and (.topic == "'$topic'")) | .flags | .[] | .name + "\t" + .description + "\t" + .type + "\t" + (.hasValue | tostring) + "\t" + .char' commands-display.json)
```

Comment:
The addition of `(.hasValue | tostring)` allows the `delimitedFlags` variable to capture when the user has entered a value in a "key=value" argument. 

Further down in the script, notable lines 92-100, the use of `$flagValue` allows the autocompletion to continue to be used after argument values have been entered for the command.

![image](https://user-images.githubusercontent.com/4142577/60752985-51ec6800-9f92-11e9-955d-ed47496d6262.png)